source/ranges.tex

```diff
@@ -2879,6 +2883,18 @@
 reads (using \tcode{operator>>}) successive elements
 from its corresponding input stream.
 
+\pnum
+The name \tcode{views::istream<T>} denotes
+a customization point object\iref{customization.point.object}.
+Given a type \tcode{T} and a subexpression \tcode{E} of type \tcode{U},
+if \tcode{U} models
+\tcode{\libconcept{derived_from}<basic_istream<typename U::char_type,
+typename U::traits_type>>},
+then the expression \tcode{views::istream<T>(E)} is expression-equivalent to
+\tcode{basic_istream_view<T, typename U::char_type,
+typename U::traits_type>(E)};
+otherwise \tcode{views::istream<T>(E)} is ill-formed.
```

Comment:
@jensmaurer I'm reinstating the comma after "otherwise"; if you have a general plan to remove such commas, let's open an issue.

Reply:
Fine with me.